### PR TITLE
[plugin][user][joomla] namespaced

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
user namespaced classes

### Testing Instructions
code review
the plugin works as before

### CODE REVIEW

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on f5fa15c55cfb1a2db0d9c7e84639a53d5f49750c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23350">issues.joomla.org/tracker/joomla-cms/23350</a>.</sub>

File: plugins/user/joomla/joomla.php [hunk 1]
```diff
@@ -9,6 +9,12 @@
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\ComponentHelper;
```
Comment (Quy) [inline plugins/user/joomla/joomla.php]:
```suggestion
use Joomla\CMS\Component\ComponentHelper;
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 4467eb0196aa1f774ab0f114256ee4a24ec1931f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23350">issues.joomla.org/tracker/joomla-cms/23350</a>.</sub>

**Comment by sandewt** [comment]
`JPlugin` could be `CMSPlugin`
see issue #23335

**Comment by AdamMadrzejewski** [comment]
I have tested this item :white_check_mark: successfully on 4467eb0196aa1f774ab0f114256ee4a24ec1931f<br /><br />I applied the path and tested it as much as I could and I think I've done everything needed for my first test. I'm going to test more issues and paths :)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23350">issues.joomla.org/tracker/joomla-cms/23350</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23350">issues.joomla.org/tracker/joomla-cms/23350</a>.</sub>

**Pull request merged**